<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="GB_DATASET_LOADER">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>381</width>
       <height>63</height>
      </rect>
     </property>
     <property name="title">
      <string>Dataset loader</string>
     </property>
     <widget class="QLineEdit" name="LE_DATASET_LOAD_PATH">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>260</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="BT_DATASET_LOAD">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GB_NETWORK_PARAMETERS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>381</width>
       <height>140</height>
      </rect>
     </property>
     <property name="title">
      <string>Network parameters</string>
     </property>
     <widget class="QLineEdit" name="LE_PARAMS_BATCH_SIZE">
      <property name="geometry">
       <rect>
        <x>129</x>
        <y>40</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_PARAMS_LAERNING_RATE">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>140</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_PARAMS_EPOCHS">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>140</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_BATCH_SIZE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Batch size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_LEARNING_RATE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Learning rate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_EPOCHS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Epochs</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="BT_LEARNING">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>89</width>
        <height>84</height>
       </rect>
      </property>
      <property name="text">
       <string>LEARNING</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="GB_MODEL_MANAGEMENT">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>381</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Model management</string>
     </property>
     <widget class="QLineEdit" name="LE_MODEL_LOAD_PATH">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>260</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="BT_MODEL_LOAD">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>LOAD</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_MODEL_SAVE_PATH">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>260</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Model.pt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BT_MODEL_SAVE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BT_MODEL_ESTIMATE">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>90</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>ESTIMATE</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="TW_LEARNING">
     <attribute name="title">
      <string>Learning</string>
     </attribute>
     <widget class="QCustomPlot" name="QCP_LEARNING" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_MINI_BATCH_LOSS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>466</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Mini-batch loss</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_LEARNING_MINI_BATCH_LOSS">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>466</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TW_ESTIMATE">
     <attribute name="title">
      <string>Estimate</string>
     </attribute>
     <widget class="QLineEdit" name="LE_MSE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>466</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>MSE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="LE_ESTIMATE_MSE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>466</y>
        <width>141</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCustomPlot" name="QCP_ESTIMATE" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
